cli/compose/types/types.go

```diff
@@ -58,6 +58,7 @@ type ConfigFile struct {
 type ConfigDetails struct {
 	Version     string
 	WorkingDir  string
+	HomeDir     string
```

Comment:
Why do we need to *pop-up* `HomeDir` in the `ConfigDetails` ? (also, it's a bit unrelated to the change as we could do the `homedir.Get` at the same point of the code where we did the `lookupEnv("HOME")`, right ?

Reply:
The home path resolution on Windows was broken (it did only do a lookupEnv("HOME")), and as we already have WorkingDir in config details, it made sense to also put HomeDir there (or remove both of them for good) for consistency towards ambient path.
Having them both in ConfigDetails makes it easier to test combinations of windows and linux base environments though.

Reply:
Yeah, I'm not a huge fan of that but… ok fair enough :stuck_out_tongue: